src/utils/generate_structs.jl

```diff
@@ -61,7 +61,7 @@ end
 {{/has_null_values}}
 {{#accessors}}
 {{#create_docstring}}\"\"\"Get {{struct_name}} {{name}}.\"\"\"{{/create_docstring}}
-{{accessor}}(value::{{struct_name}}) = value.{{name}}
+{{accessor}}(value::{{struct_name}}) = {{#needs_conversion}}get_value(value, :{{name}}){{/needs_conversion}}{{^needs_conversion}}value.{{name}}{{/needs_conversion}}
```

Comment:
What do you think about changing `needs_conversion` to `conversion_function` and then replace `get_value` with that?  Then the user could specify any custom function.